packages/clerk-js/src/ui/common/PrintableComponent.tsx

```diff
@@ -13,7 +13,7 @@ export const usePrintable = (): UsePrintableReturn => {
   return { print, printableProps: { onPrint } };
 };
 
-export const PrintableComponent = (props: UsePrintableReturn['printableProps'] & React.PropsWithChildren<{}>) => {
+export const PrintableComponent = (props: UsePrintableReturn['printableProps'] & React.PropsWithChildren) => {
```

Comment:
For every `React.PropsWithChildren` instance I removed the unnecessary generic because it already has a default. So we don't need to set `{}`

Reply:
For anyone wondering, as of React 18, `PropsWithChildren` uses a default type param and is defined as: 
```ts
    type PropsWithChildren<P = unknown> = P & { children?: ReactNode | undefined };
```